dev/tests/integration/testsuite/Magento/ProductAlert/_files/customer_unsubscribe_stock_rollback.php

```diff
@@ -0,0 +1,20 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+use Magento\Catalog\Api\ProductRepositoryInterface;
+use Magento\ProductAlert\Model\ResourceModel\Stock;
+use Magento\TestFramework\Helper\Bootstrap;
+
+$objectManager = Bootstrap::getObjectManager();
+$resource = $objectManager->get(Stock::class);
+
+$productRepository = $objectManager->create(ProductRepositoryInterface::class);
```

Comment:
Why are you using `create`?

Reply:
Changed to 'get'. Should have been that way initially, result of copypasting